maintenance/image-options/imageset.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Deploy images by digest
+description: Specify the digests the operator will use to deploy images.
+---
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Big picture
+
+Enable deploying images by container registry digest for operator installations.
```

Comment:
Given change to above verb, I would change this as well to:

Deploy images by container registry digest for operator installations.